chain/ton/provider/rpc_provider_test.go

```diff
@@ -72,135 +72,153 @@ func Test_RPCChainProviderConfig_validate(t *testing.T) {
 	}
 }
 
-func Test_RPCChainProvider_Initialize(t *testing.T) {
+func Test_RPCChainProvider_Initialize_AlreadyInitialized(t *testing.T) {
```

Comment:
```suggestion
func Test_RPCChainProvider_Initialize(t *testing.T) {
```
why is this call AlreadyInitialized? Seems to be only initialized once?